docs/release-notes/8.0/8.0.35-30.0.md

```diff
@@ -0,0 +1,25 @@
+# Percona XtraBackup 8.0.35-30 (2023-11-)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+## Release highlights
+
+This release merges the MySQL 8.0.35 code base.
+
+## Bug fix
+
+* [PXB-3003](https://jira.percona.com/browse/PXB-3003): The `log_files_find_and_analyze` option had a potential race condition when the user listed the available files and tried to open them. In between these two operations, the server could delete the log files, causing the `log_files_find_and_analyze` option to return an error.
```

Comment:
This is actually nothing to do with user options

Reply:
Add something like:

A possible race conditing while PXB is discovering redo logs to parse and server purging them.

Reply:
A race condition could occur when Percona XtraBackup discovers redo logs to parse and the server purges redo logs at the same time.

Reply:
fixed